airbyte/_executors/util.py

```diff
@@ -25,56 +25,50 @@
     from airbyte._executors.base import Executor
 
 
-def _try_get_source_manifest(source_name: str, manifest_url: str | None) -> dict:
+VERSION_LATEST = "latest"
+DEFAULT_MANIFEST_URL = (
+    "https://storage.googleapis.com/prod-airbyte-cloud-connector-metadata-service/"
+    "metadata/airbyte/{source_name}/{version}/manifest.yaml"
+)
```

Comment:
@bnchrch - Instead of the public GCS bucket endpoint, should I use the Airbyte one:

E.g. `https://connectors.airbyte.com/files/metadata/airbyte/source-the-guardian-api/0.2.0/manifest.yaml](http://connectors.airbyte.com/files/metadata/airbyte/source-the-guardian-api/0.2.0/manifest.yaml`

Reply:
@bnchrch - Done ✅ 

I also just added a user agent string so we can see how many calls to those endpoint come from PyAirbyte and which version of PyAirbyte is being used.